docs/team/adding-members.rst

```diff
@@ -0,0 +1,108 @@
+======================================
+Adding and onboarding new team members
+======================================
+
+This page contains information about adding new team members and onboarding
+them.
+
+Adding a new team members
+=========================
+
+For someone to become a team member, they should already be a consistent,
+positive, productive member of the community. Newcomers are encouraged to
+become team members, but after they've shown a sustained interest in
```

Comment:
```suggestion
become team members, after they've shown a sustained interest in
```
feels like with the "but" there we'd have to make it "but only". I prefer it like this